src/abft-messages-proposals.md

```diff
@@ -1,30 +1,46 @@
+$$
+\newcommand \pk {\mathrm{pk}}
+\newcommand \fv {\text{first}}
+\newcommand \lv {\text{last}}
+\newcommand \Sign {\mathrm{Sign}}
+\newcommand \ValidEntry {\mathrm{ValidEntry}}
+\newcommand \Hash {\mathrm{Hash}}
+\newcommand \Digest {\mathrm{Digest}}
+\newcommand \Encoding {\mathrm{Encoding}}
+\newcommand \Record {\mathrm{Record}}
+\newcommand \Verify {\mathrm{Verify}}
+\newcommand \Proposal {\mathrm{Proposal}}
+$$
+
 # Proposals
 
-Let \\(e = (o, Q)\\) be an entry and \\(y\\) be the output of a \\(Sign\\) procedure.
+Let \\( e = (o, Q) \\) be an entry and \\( y \\) be the output of a \\( \Sign \\)
```

Comment:
@cce @jannotti the seed symbol has been renamed from s (original spec) to Q, since it is the symbol usually used for the seed.